### PR TITLE
Fix hypertable foreign keys for 2.15.X

### PR DESCRIPTION
**Pull request by fabriziomello**

### CODE REVIEW

**Review by svenklemm — APPROVED** [review]

File: utils/2.15.X-fix_hypertable_foreign_keys.sql [hunk 1]
```diff
@@ -0,0 +1,77 @@
+CREATE OR REPLACE FUNCTION pg_temp.constraint_columns(regclass, int2[]) RETURNS text[] AS
```
Comment (svenklemm) [inline utils/2.15.X-fix_hypertable_foreign_keys.sql:1]: -- Fix compressed hypertables with FOREIGN KEY constraints that were created with TimescaleDB versions before 2.15.0

Comment (svenklemm) [inline utils/2.15.X-fix_hypertable_foreign_keys.sql:1]:
```suggestion
-- Fix compressed hypertables with FOREIGN KEY constraints that were created with TimescaleDB versions before 2.15.0

CREATE OR REPLACE FUNCTION pg_temp.constraint_columns(regclass, int2[]) RETURNS text[] AS
```

**Review by svenklemm — APPROVED** [review]

File: utils/2.15.X-fix_hypertable_foreign_keys.sql (hunk 1 above)
Comment (svenklemm) [reply utils/2.15.X-fix_hypertable_foreign_keys.sql:1]: a comment with short explanation cant hurt

File: utils/2.15.X-fix_hypertable_foreign_keys.sql [hunk 2]
```diff
@@ -0,0 +1,79 @@
+-- Fix compressed hypertables with FOREIGN KEY constraints that were created with TimescaleDB versions before 2.15.0
+
```
Comment (fabriziomello) [inline utils/2.15.X-fix_hypertable_foreign_keys.sql:2]:
```suggestion
```

**Pull request merged**